cmd/validate-krew-manifest/main.go

```diff
@@ -197,3 +208,26 @@ func allPlatforms() [][2]string {
 		{"darwin", "amd64"},
 	}
 }
+
+// validateLineLength checks if the maximum line length is below maxLen.
+// Lines may exceed this limit, if they are just one word, i.e. without spaces.
+func validateLineLength(s string, maxLen int) bool {
+	for _, line := range strings.Split(s, "\n") {
```

Comment:
how about carriage return `\r`?

Reply:
It should not happen, but better safe than sorry :)